src/bridge-deployer/README.md

```diff
@@ -53,15 +56,38 @@ For the deployment of canisters, you will require to have/create a wallet canist
 
 For the deployment, you will need to provide the wallet canister id as `--wallet-canister` or as an environment variable `WALLET_CANISTER`.
 
-Command to deploy a bridge canister:
+Command to deploy a bridge canister connecting the bridge to the EVM canister:
 
 ```bash
 ./bridge-deployer
   -vvv \
-  --evm-network localhost \
+  --ic-network localhost \
   --private-key <PRIVATE_KEY> \
   --identity path/to/identity.pem \
-  --evm <EVM_PRINCIPAL> \
+  --evm-principal <EVM_PRINCIPAL> \
+  deploy \
+  --wasm path/to/rune_bridge.wasm \
+  --wallet-canister <WALLET_CANISTER> \
+  rune \
+  --owner <ADMIN_PRINCIPAL> \
+  --min-confirmations 6 \
+  --indexer-urls <https://indexer1.com,https://indexer2.com,https://indexer3.com> \
+  --deposit-fee 1000000 \
+  --mempool-timeout 3600 \
+  --signing-key-id dfx \
+  --bitcoin-network <bitcoin_network> \
+  --indexer-consensus-threshold 3
```

Comment:
When I wrote this documenration, my logic was to give here the smallest possible example that would work with default parameters. Given an example with all possible configuration is usually very scary for new users. They can find all the options with much more information in `bridge-deployer --help`...